source/OptimaJet.DWKit.StarterApplication/BackendServicesExtension.cs

```diff
@@ -81,6 +81,9 @@ public static IServiceCollection AddBackgroundServices(this IServiceCollection s
             services.AddFluentEmail(GetVarOrDefault("ADMIN_EMAIL", "noreply@scriptoria.io"), GetVarOrDefault("ADMIN_NAME", "Scriptoria Mailer"))
                     .AddRazorRenderer();
 
+            services.AddScoped<JobProjectRepository>();
+            services.AddScoped<JobOrganizationRepository>();
+            services.AddScoped<JobSystemStatusRepository>();
             services.AddScoped(typeof(IJobRepository<>), typeof(JobRepository<>));
             services.AddScoped<IJobRepository<Email>, JobEmailRepository>();
```

Comment:
Do we need concrete classes for JobProjectRepository, JobOrganizationRepository, and JobSystemStatusRepository (since we aren't overriding anything), or can just use IJobRepository<Project>, IJobRepository<Organization> and IJobRepository<SystemStatus> in constructors (dependency injection)?